lib/rules/rules.json

```diff
@@ -34,6 +38,10 @@
         "alternative": "Array.prototype.find()",
         "ES6": true
     },
+    "first": {
+        "compatible": true,
+        "alternative": "Array[0] or Array.prototype.slice()"
```

Comment:
This should just be `Array.prototype.slice()`

Reply:
Updated